v19.1/build-a-c++-app-with-cockroachdb.md

```diff
@@ -12,15 +12,32 @@ We have tested the [C++ libpqxx driver](https://github.com/jtv/libpqxx) enough t
 
 ## Before you begin
 
-Make sure you have already [installed CockroachDB](install-cockroachdb.html).
+{% include {{page.version.version}}/app/before-you-begin.md %}
 
 ## Step 1. Install the libpqxx driver
 
 Install the C++ libpqxx driver as described in the [official documentation](https://github.com/jtv/libpqxx).
 
-{% include {{ page.version.version }}/app/common-steps.md %}
+{{site.data.alerts.callout_info}}
+If you are running macOS, you need to install version 4.0.1 of the libpqxx driver.
```

Comment:
Perhaps this should say "4.0.1 or higher" ?  I was able to use the one currently available from Homebrew, which is 6.4.5, and it worked great.

Reply:
Yes, I will do that. I installed 4.0.0 for some reason(?), and it didn't have this mac-specific bug fixed yet: https://github.com/Homebrew/legacy-homebrew/issues/23627. Your language is more accurate.